tests/test_sync.py

```diff
@@ -338,3 +372,63 @@ def test_synchronize_with_cleanup_cli():
     assert "src/ansys/tools/repo_sync/__init__.py" in captured
     assert "src/ansys/tools/repo_sync/__main__.py" in captured
     assert "src/ansys/tools/repo_sync/repo_sync.py" in captured
+
+
+@pytest.mark.skipif(SKIP_LOCALLY, reason="Only runs on workflow")
+def test_synchronize_with_no_sync_cli():
+    """Test synchronization tool (with no files needed to be synced) from CLI."""
+
+    # Define a temp directory and copy assets in it
+    temp_dir = tempfile.TemporaryDirectory(prefix="repo_clone_cli_")
+    shutil.copytree(
+        ASSETS_DIRECTORY,
+        temp_dir.name,
+        dirs_exist_ok=True,
+    )

```

Comment:
Can the `tmpfile` fixture be used for this? https://docs.pytest.org/en/6.2.x/tmpdir.html